src/components/LatestBlogs/index.tsx

```
@@ -0,0 +1,61 @@
+          <BlogCard {...blog} key={index} />
+        ))}
+
+      </div>
+
+      <div className="text-center">
+        <div className="inline-block bg-white border border-gray-200 shadow-2xs rounded-full dark:bg-neutral-900 dark:border-neutral-800">
+          <div className="py-3 px-4 flex items-center gap-x-2">
+            <p className="text-gray-600 dark:text-neutral-400">
+              Want to read more?
+            </p>
+            <a className="inline-flex items-center gap-x-1.5 text-blue-600 decoration-2 hover:underline focus:outline-hidden focus:underline font-medium dark:text-blue-500" href="#">
```

Biome (2.1.2)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)